src/labels_by_user_input.py

```diff
@@ -4,18 +4,27 @@
 LABEL_VERIFIED = "Verified"
 
 
+def remove_label(pull, label):
+    print(f"Remove {label} from {pull.title}")
+    pull.remove_from_labels(label)
+
+
 def remove_verified_label(pull):
     labels = get_labels(pull=pull)
     if LABEL_VERIFIED in labels:
-        print(f"Remove {LABEL_VERIFIED} from {pull.title}")
-        pull.remove_from_labels(LABEL_VERIFIED)
+        remove_label(pull=pull, label=LABEL_VERIFIED)
 
 
 def labels_by_user_input(data, pull):
     body = data["comment"]["body"]
-    if "/verified" in body and LABEL_VERIFIED not in get_labels(pull=pull):
-        print(f"Adding {LABEL_VERIFIED} to {pull.title}")
-        pull.add_to_labels(LABEL_VERIFIED)
+    labels = get_labels(pull=pull)
+
+    if body[0] == "/":
```

Comment:
I don't think we want to add any label
We need to define the one that we want
for example:
/hold >> do-not-merge/on-hold

Reply:
We need to be able to find the string anywhere in the body.
I can do something like:

all tests passed on my local cluster
/verified